Dezenix Info card/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>DEZENIX | Info card</title>
+    <link rel="stylesheet" href="style.css">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Bangers&family=Cinzel:wght@500&family=Macondo&family=Roboto:ital,wght@0,100;1,100&family=Skranji&family=Smooch&display=swap"
```

Comment:
Change the body font, it is not making sense in capital

Reply:
Hey Ayush ... Do you want me to change the font or do you want me to make the letters in small case?